leases/models.py

```diff
@@ -218,8 +228,12 @@ def clean(self):
             existing_leases = existing_leases.exclude(pk=self.pk)
         if existing_leases.exists():
```

Comment:
With the current implementation there can be only one "winterstorage area type lease", if you try to add another one, you get "WinterStoragePlace already has a lease" because of this check.

Reply:
Ohhhhh sneaky one. 😅

Reply:
There seems to be still problem with this :)
````
        elif self.area:
            existing_leases = existing_leases.exclude(area=self.area)
````
With the check above you get "WinterStoragePlace already has a lease" if some *other* WS Area has leases.

Reply:
Right 🤔 then if the lease is associated with an area, we don't need to check fro `existing` leases since there can be many associated to the it.